authors/apps/authentication/migrations/0001_initial.py

```diff
@@ -1,41 +0,0 @@
-# Generated by Django 2.1.7 on 2019-03-19 12:02
-
-from django.db import migrations, models
```

Comment:
@ManuelDominic, any reason you are removing this migration

Reply:
@kimbugp Thanks for the feedback, error fixed.